client/src/legacy/TinyMCE_sslink-internal.js

```diff
@@ -0,0 +1,128 @@
+/* global tinymce, ss */
+import i18n from 'i18n';
+import TinyMCEActionRegistrar from 'lib/TinyMCEActionRegistrar';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ApolloProvider } from 'react-apollo';
+import jQuery from 'jquery';
+import ShortcodeSerialiser from 'lib/ShortcodeSerialiser';
+import { createInsertLinkModal } from 'containers/InsertLinkModal/InsertLinkModal';
+import { provideInjector } from 'lib/Injector';
+
+// Link to external url
+TinyMCEActionRegistrar.addAction('sslink', {
+  text: i18n._t('CMS.LINKLABEL_PAGE', 'Link to page'),
+  // eslint-disable-next-line no-console
+  onclick: (editor) => editor.execCommand('sslinkinternal'),
+});
+
+const plugin = {
+  init(editor) {
+    editor.addCommand('sslinkinternal', () => {
+      const field = $(`#${editor.id}`).entwine('ss');
```

Comment:
Still similar issue as before. `$` here might be different from the imported `jQuery`.